### PR TITLE
ompgsql - fix unsafe string copy in defaults

### PR DESCRIPTION
**Pull request by GrayTShirt**

### CODE REVIEW

**Comment by GrayTShirt** [comment]
looks like raspbian failure is unrelated.

**Comment by rgerhards** [comment]
yeah, keep cool ;-) I'll follow these things, especially as I know some parts of the CI are still somewhat fragile (especially the newly added raspberry). Actually, I've already resubmitted the build, but it's right now in the queue. If an issue comes up that looks unrelated, just wait a litle but, I'll babysit them (except if I forget ;-)).

File: plugins/ompgsql/ompgsql.c [hunk 1]
```diff
@@ -363,8 +363,8 @@ setInstParamDefaults(instanceData *pData)
 	pData->trans_commit  = 100;
 	pData->trans_age     = 60;
 	pData->port          = 5432;
-	strncpy(pData->user, "postgres", strlen("postgres"));
-	strncpy(pData->pass, "postgres", strlen("postgres"));
+	strncpy(pData->user, "postgres", strlen("postgres") + 1);
```
Comment (rgerhards) [inline plugins/ompgsql/ompgsql.c]: thinking twice... shouldn't that be ```..., sizeof(pData->user));``` as ```strncpy()``` intends to guard that the buffer is not overrun?

Comment (dmolik) [reply plugins/ompgsql/ompgsql.c]: That makes sense.

**Comment by rgerhards** [comment]
raspbian is having some strange issue... This PR is fine for sure, I merge it now despite raspbian.

**Pull request merged**